ROADMAP.md

```diff
@@ -57,4 +57,14 @@ NeoGo retains certain deprecated error codes: `neorpc.ErrCompatGeneric`,
 `neorpc.ErrCompatNoOpenedWallet`. They returned by nodes not compliant with the 
 neo-project/proposals#156 (NeoGo pre-0.102.0 and all known C# versions).
 
-Removal of the deprecated RPC error codes is planned once all nodes adopt the new error standard.
\ No newline at end of file
+Removal of the deprecated RPC error codes is planned once all nodes adopt the new error standard.
+
+## NotaryServiceFeePerKey native Notary contract functionality removal
+
+`getNotaryServiceFeePerKey` and `setNotaryServiceFeePerKey` methods were moved from
+native Notary contract to the native PolicyContract contract to be a part of more
```

Comment:
@roman-khimov, I'm not sure, do we really need to keep these old APIs? The contract is not yet a part of the core protocol. so maybe it's OK to remove it instantly?

Reply:
I doubt anyone has ever used them and your changes basically deactivate them, so they can be removed immediately to prevent any new uses.